http/handlers.go

```diff
@@ -163,11 +163,11 @@ func ErrorHandler(logger log.Logger) func(context.Context, error, http.ResponseW
 		switch e := errors.Cause(err).(type) {
 		case security.ForbiddenError:
 			w.WriteHeader(http.StatusForbidden)
-			w.Write([]byte(errorhandler.GetEmitter() + "." + errorhandler.MsgErrOpNotPermitted))
+			_, _ = w.Write([]byte(errorhandler.GetEmitter() + "." + errorhandler.MsgErrOpNotPermitted))
```

Comment:
?

Reply:
Reverted to keep sonar code smell and process it correctly when we'll have time